lms/static/js/learner_dashboard/views/program_details_view.js

```diff
@@ -29,6 +43,16 @@
                     this.headerView = new HeaderView({
                         model: this.programModel
                     });
+                    new CollectionListView({
+                        el: '.js-course-list',
+                        childView: CourseCardView,
+                        collection: this.courseCardsCollection,
+                        context: this.context,
```

Comment:
Instead of context you could pass ```this.model.toJSON()``` or a subset of the model attributes with just the expected object.